fleet-secrets-bro-patch/patch_gitrepo_secrets.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+DRYRUN=0
+
+while getopts "D:" opt; do
```

Comment:
```suggestion
while getopts "D" opt; do
```

Remove trailing `:` so an argument is not expected